t/ui/10-tests_overview.t

```diff
@@ -543,18 +543,34 @@ subtest 'filter by result and state' => sub {
     ok !$driver->find_element_by_id('filter-failed')->is_selected, 'other checkbox not checked';
 };
 
+subtest "job template names displayed on 'Test result overview' page" => sub {
+    $driver->get('/group_overview/1002');
+    is($driver->find_element('.progress-bar-unfinished')->get_text(),
+        '1 unfinished', 'The number of unfinished jobs is right');
```

Comment:
We normally start test names with lower case. You can also drop the "is right" in the end which is kind of a given for checks:
```suggestion
        '1 unfinished', 'expected number of unfinished jobs');
```

EDIT: I now see that you're just moving this code. My suggestion would nevertheless be a good improvement.

Reply:
@b10n1k How about this one?

Reply:
Not changed in the latest diff. Ofc you can argue that you don't want to change it. Be explicit, though, rather than leaving questions unanswered.

Reply:
Done